src/client-entry.ts

```diff
@@ -5,14 +5,16 @@ import TermboxRequest from '@/common/TermboxRequest';
 import { factory } from '@/common/TermboxFactory';
 import UlsLanguageTranslationRepository from '@/client/data-access/UlsLanguageTranslationRepository';
 import EntityRepository from '@/client/data-access/EntityRepository';
-import MwWindow from './client/MwWindow';
+import MwWindow from '@/client/mediawiki/MwWindow';
```

Comment:
I vaguely remember the discussion when we introduced this. 
I'm still fine with inline casting when using e.g. window in a way that we "hope it to be" a MwWindow.
But this move into a dedicated folder makes me question why this is not in a (obviously client-only) `.d.ts` folder instead...